AirCasting/SDSync/ViewsFlow/SDRestartAB/SDRestartABView.swift

```diff
@@ -54,15 +57,15 @@ extension SDRestartABView {
     
     var continueButton: some View {
         Button {
-            viewModel.presentNextScreen = true
+            viewModel.presentNextScreen.toggle()
```

Comment:
Why did you change "= true" to toggle?

Reply:
It seems better/cleaner in my opinion and work as expected :)